burst/src/test/java/com/squareup/burst/BurstTest.java

```diff
@@ -1,10 +1,12 @@
 package com.squareup.burst;
 
-import java.lang.reflect.Method;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import java.lang.annotation.Annotation;
```

Comment:
nit: our preferred [import layout](https://github.com/square/java-code-styles/blob/master/configs/codestyles/Square.xml#L21) is all non-static imports together, a blank line, then all static imports.